addons/tls13-compat-ff51/bootstrap.js

```diff
@@ -150,28 +183,31 @@ function install() {
     console.log("User has changed TLS max version. Skipping");
     return;
   }
-  
+
+  // Only ever want to run once.
+  gNewInstall = true;
+
```

Comment:
As noted in the latest commit, I re-tested and can't reproduce this issue so I put the `setTimeout` back into `install()`